language/documentation/book/translations/zh/abilities.md

```diff
@@ -34,27 +34,27 @@ The four abilities are:
 
 ### `copy`
 
-The `copy` ability allows values of types with that ability to be copied. It gates the ability to copy values out of local variables with the [`copy`](./variables.md#move-and-copy) operator and to copy values via references with [dereference `*e`](./references.md#reading-and-writing-through-references).
+The `copy` ability allows values of types with that ability to be copied. It gates the ability to copy values out of local variables with the [`copy`](variables.md#move-and-copy) operator and to copy values via references with [dereference `*e`](references.md#reading-and-writing-through-references).
```

Comment:
`./` should not be removed, otherwise GitHub will not be able to index the file.
![B046599B-B795-41EE-92A9-54A229A81E06](https://user-images.githubusercontent.com/54882546/194739150-7f9808b2-f8ec-41ab-8820-8221b2f5d8fb.jpeg)

Reply:
Hi, thanks for pointing it out. I've tried and met the same problem on my local-md editor.

The reason is not about the `./`, It's about the referred title `#blabla` has been translated into Chinese.

However, It works fine on the [docs-site](https://chengyuejia.github.io/move/zh/abilities.html). 

I've fix the this one.

Due to the translation is undone, there still are related bugs like this.
 @lshoo @geometryolife Hope we can fix them together later.

This pr is for enhance the i18n ability for move-book. 🐶

Reply:
So, I think it needs to be changed back to `./`, which guarantees better compatibility.

Reply:
I've fixed it. And resolve the conflicts